helm/defectdojo/templates/configmap.yaml

```diff
@@ -38,6 +38,7 @@ data:
   DD_UWSGI_PASS: unix:///run/defectdojo/uwsgi.sock
   DD_UWSGI_NUM_OF_PROCESSES: '{{ .Values.django.uwsgi.app_settings.processes | default 2 }}'
   DD_UWSGI_NUM_OF_THREADS: '{{ .Values.django.uwsgi.app_settings.threads | default 2 }}'
+  DD_UWSGI_MAX_FD: '{{ .Values.django.uwsgi.app_settings.max_fd }}'
```

Comment:
Could you add this new value to the `values.yaml` file so people know it's an available option? I'm not sure what the right default value would be, so if you want you can just leave it commented out to start with.

Reply:
This is fixed now. Have requested a new review